homeassistant/config_entries.py

```diff
@@ -1814,6 +1814,8 @@ async def async_forward_entry_setups(
         self, entry: ConfigEntry, platforms: Iterable[Platform | str]
     ) -> None:
         """Forward the setup of an entry to platforms."""
+        integration = await loader.async_get_integration(self.hass, entry.domain)
```

Comment:
Would it be interesting to explore being able to pass platforms we expect to load to this function too?

Reply:
We could add a list of platforms to preload in the manifest. That would allow us to do it when the component is loaded but it's such a rare case where that's desirable if it's not a stage 1 or before integration

Reply:
Oh right, by the time we forward entries, we've already loaded the integration.